lib/foreman_theme_satellite/documentation.rb

```diff
@@ -80,5 +80,11 @@ module Documentation
     SPECIAL_LINKS = [
```

Comment:
Technically we can drop these now, as the only "special" link is now handled by the "real" docs controller.